interfaces.ts

```diff
@@ -51,6 +58,13 @@ export interface Infotag {
   infotag_group: number;
 }
 
+/**
+ * Public character page. Why does this extend SimpleCharacter? Investigate usage.
```

Comment:
I'm going to assume this extends SimpleCharacter because of something that was designed for the old 2.0 site.